phpstan.neon

```diff
@@ -4,6 +4,8 @@ includes:
 parameters:
   level: 7
   reportUnmatchedIgnoredErrors: false
+  checkMissingIterableValueType: false
+  checkGenericClassInNonGenericObjectType: false
```

Comment:
This are new options coming with the new version of PHPStan. They indroduce new checks, which we didn't have in previous versions, so I disabled them.